test/rekt/resources/eventcache/README.md

```diff
@@ -0,0 +1,20 @@
+# Event Cache
```

Comment:
Do we need this in eventing?

Why not part of the `rekt` framework?

Reply:
it needs to be local to build the kodata file.